### PR TITLE
fix compatibility for v0.4 , v0.5, v0.6

### PR DESCRIPTION
**Pull request by jlapeyre**
Fixes for versions of Julia.

### CODE REVIEW

**Comment by jlapeyre** [comment]
added julia 0.4 to REQUIRE to fix Travis build failure

File: LambertW/versions/0.0.5/requires [hunk 1]
```diff
@@ -0,0 +1 @@
+Compat
```
Comment (ararslan) [inline LambertW/versions/0.0.5/requires]: This file should include a minimum Julia version

File: ZChop/versions/0.0.3/requires [hunk 2]
```diff
@@ -0,0 +1 @@
+Compat
```
Comment (ararslan) [inline ZChop/versions/0.0.3/requires]: This too

Comment (jlapeyre) [inline ZChop/versions/0.0.3/requires]: My METADATA repo  is in a state of confusion. I will close this PR, fix the repo and submit an new PR.

**Pull request closed without merging**